contributors/devel/sig-architecture/api_changes.md

```diff
@@ -939,7 +939,7 @@ an example to illustrate your change.
 Make sure you update the swagger and OpenAPI spec by running:
 
 ```sh
-hack/update-swagger-spec.sh
+hack/update-generated-swagger-docs.sh
 hack/update-openapi-spec.sh
```

Comment:
```suggestion
make update
```

Let's just use the generic command, as this may change in the future, but the make target is well known